{"name": "test_last_heading", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestSignUp' object has no attribute 'driver'", "trace": "self = <signup_test.TestSignUp object at 0x000001FAE0B5D9A0>\n\n    @pytest.fixture(scope='function', autouse=True)\n    def signup_fixture(self):\n>       self.sign = SignUp_Page(self.driver)\nE       AttributeError: 'TestSignUp' object has no attribute 'driver'\n\ntests\\signup_test.py:18: AttributeError"}, "description": "Check the SignUp Page", "start": 1730126547815, "stop": 1730126547815, "uuid": "a5b04637-0e2c-469b-ad0b-ada1bfde33f4", "historyId": "ecc9d9ebe69040fcb3b599e5c7b02e2d", "testCaseId": "ecc9d9ebe69040fcb3b599e5c7b02e2d", "fullName": "tests.signup_test.TestSignUp#test_last_heading", "labels": [{"name": "severity", "value": "critical"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "signup_test"}, {"name": "subSuite", "value": "TestSignUp"}, {"name": "host", "value": "MSI"}, {"name": "thread", "value": "29152-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.signup_test"}]}